app/core/models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+
+
+class Ingredient(models.Model):
+    """
+    Ingredient used in a recipe
+    """
+    name = models.CharField(max_length=255)
+    recipe = models.ForeignKey(
+        'Recipe',
+        on_delete=models.CASCADE,
+        related_name='ingredients'
+    )
+
+    def __str__(self):
+        return self.name
+
+
+class Recipe(models.Model):
+    """
+    Recipe object representation
+    """
+    name = models.CharField(max_length=255)
+    description = models.CharField(max_length=400)
```

Comment:
400 characters for a recipe description is probably too short. I would put either a [`TextField`](https://docs.djangoproject.com/en/3.0/ref/models/fields/#textfield) here or a longer length.

Reply:
Changed